pkgs/misc/vere/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv
+, lib
+, fetchzip
+}:
+
+let
+  os = if stdenv.isDarwin then "macos" else "linux";
+  arch = if stdenv.isAarch64 then "aarch64" else "x86_64";
+  platform = "${os}-${arch}";
+in
+stdenv.mkDerivation rec {
+  pname = "vere";
+  version = "1.20";
+
+  src = fetchzip {
+    url = "https://github.com/urbit/vere/releases/download/vere-v${version}/${platform}.tgz";
+    sha256 = {
+      x86_64-linux = "sha256-nBIpf9akK4cXnR5y5Fcl1g7/FxL8BU/CH/WHGhYuP74=";
+      aarch64-linux = "sha256-ERSYXNh/vmAKr4PNonOxTm5/FRLNDWwHSHM6fIeY4Nc=";
+      x86_64-darwin = "sha256-Kk9hNzyWngnyqlyQ9hILFM81WVw1ZYimMj4K3ENtifE=";
+      aarch64-darwin = "sha256-i3ixj04J/fcb396ncINLF8eYw1mpFCYeIM3f74K6tqY=";
+    }.${stdenv.hostPlatform.system} or (throw "unsupported system ${stdenv.hostPlatform.system}");
+  };
```

Comment:
It isn't a deal-breaker, but ideally, the we want to build stuff from source for multiple reasons:

1. we can support patching at the packaging and user levels.
2. it creates a more software supply chain that is more easily inspected

I'm not super familiar with building bazel packages in Nix, so I'm not sure how much work will be required to make it happen.

Reply:
Awesome, thanks! Yes, I agree w.r.t. building from source-- I tried my best last week to get our `bazel` commands working within `nix` but failed and had to switch to another project after half a day. Seeing how `buildBazelPackage` was used in other derivations helped, but ultimately I wasn't able to configure a working derivation to build `urbit` from source. For getting a working derivation released sooner rather than later, I hope using the binaries suffices for now. I'd be happy to work on getting a proper build-from-source derivation working in the future, though.

Also, thanks for the `updateScript` suggestion-- I just pushed one.